spaghetti-core/src/main/ts/tsAstParser.ts

```diff
@@ -239,14 +293,26 @@ if (args[0] === "--collectExportedIdentifiers") {
         process.exit(0);
     }
 } else if (args[0] === "--verifyCommonJsModuleDefinition") {
```

Comment:
You should remove the whole `--verifyCommonJsModuleDefinition` because it's not used anymore.